src/client/components/componentWithSliders/allSliders/allBlocks.js

```diff
@@ -0,0 +1,73 @@
+import React, {Component} from 'react';
+import './alllBlocks.css';
+
+import BlockOfSliders from './blockOfSliders';
+import PlayButton from './playButton';
+import StopButton from './stopButton';
+
+import BLOCKS from '../../../helpers/blocks';
+import Pizzicato from "pizzicato";
+
+
+class AllBlocks extends Component {
+
+  constructor(props, context){
+    super(props, context);
+    this.state = {
+      sound: new Pizzicato.Sound({
```

Comment:
Please, fix the code formatting from 17 line till 26

Reply:
fixed